index.html

```diff
@@ -11,7 +11,37 @@
     </head>
 
     <body>
-        <main class="tableContainer"></main>
+        
+
+        <section class="controlContainer">
+
+            <div class="buttonContainer">
+                <button class="reproduce">Reproduce</button>
+                <button class="clear">Clear</button>
+                <button class="random">Random</button>
+                <button class="reset">Reset</button>
+            </div>
+    
+            <div class="sliderContainer">
+                <div class="sliderContainer_columns">
+                    <p>
+                        <p>Grid size: <span class="columns">30</span> blocks tall and large </p>
+                    </p>
+                    <input type="range" min="5" max="64" value="30" id="sliderColumns" class="slider">
```

Comment:
Hi Rodrigo G. I think that in line #30 we can delate the id="sliderColumns" and work that input only with the class.

Reply:
You right, will change it right away.
Ty for letting me know.